guides/getting-started/set-up/anoncreds-rs.md

```diff
@@ -0,0 +1,126 @@
+# Anoncreds-rs
+
+[AnonCreds](https://github.com/hyperledger/anoncreds) is a direct
+implementation of the [AnonCreds V1.0
+specification](https://hyperledger.github.io/anoncreds-spec/) that provides
+functionality like; creating a schema object, creating a credential definition
+object, creating a credential, verifying a proof presentation and much more.
```

Comment:
Although it works in the rendered html, adding a line breaks, breaks content in the preview of other pages when linked.

(see image in PR review message)